app/src/processing/app/contrib/Contribution.java

```diff
@@ -321,4 +321,23 @@ static private String translateCategory(String cat) {
     String cleaned = cat.replaceAll("[\\W]+", "_").toLowerCase();
     return Language.text("contrib.category." + cleaned);
   }
+  
+  String getPrettyVersionShort() {
+
+    String latestVersion = this.getPrettyVersion();
+    if (latestVersion != null && !latestVersion.isEmpty()) {
+      if (latestVersion.endsWith("alpla")) {
```

Comment:
Let's not get into parsing/cleaning what other people have put into the fields. We should contact library authors and have them fix the files (which can be done quickly), not change code that might stick around for years.